inventory/src/main/java/org/flyve/inventory/categories/CategoryValue.java

```diff
@@ -165,4 +166,9 @@ public Category getCategory() {
     public List<String> getValues() {
         return values;
     }
+
+    private boolean hasCharToReplace(final String val)
```

Comment:
Nice 👍 
But use braces the correct way, for example:
Correct:
try {
...
Instead of:
try
{